### DISCUSSION TITLE
Where is the data persistence directory for BanyanDB?

### DISCUSSION THREAD

**Discussion by forgottener** [apache/skywalking#12480] [Q&A] [answered]
I am currently using the Docker image `apache/skywalking-banyandb:0.6.1`. The data is lost after the Docker container restarts. Could you please tell me what the data directory for BanyanDB is?

**Comment by hanahmily** [comment, marked as answer]
Please issue the following command to run the apache/skywalking-banyandb:0.6.1 standalone: `docker run apache/skywalking-banyandb:0.6.1 standalone -h`. You need to specify the import paths for the data storage using `--measure-root-path`, `--stream-root-path`, and `--metadata-root-path` options. 

The simplest way to persist data is to mount a volume to `/tmp`, where these paths default to reside.
